Build.bee.cs

```diff
@@ -0,0 +1,377 @@
+                {"LIB", windowsSdk.LibraryPaths
+                    .Select(p => p.ResolveWithFileSystem().MakeAbsolute().ToString(SlashMode.Native))
+                    .SeparateWith(";")}
+            };
+        }
+        else if (programConfiguration.Sdk is MacSdk)
+        {
+            MacSdk macSdk = (MacSdk)programConfiguration.Sdk;
+
+            inputs = new[] {
+                macSdk.BinPath,
+                macSdk.SysRoot
+            };
+
+            var compileFlags = new[] {
+                $"-mmacosx-version-min={ChooseMinMacOSVersion(macSdk.Architecture)}"
+            };
+
+            commandLineArguments.Add($"CXXFLAGS=\"{string.Join(" ", compileFlags)}\"");
+
+            environmentVariables = new Dictionary<string, string> {
+                {"PATH", (new[] {macSdk.BinPath})
+                    .Concat(path)
+                    .Select(p => p.ResolveWithFileSystem().MakeAbsolute().ToString(SlashMode.Native))
+                    .SeparateWith(":")},
+                {"SDKROOT", macSdk.SysRoot.ResolveWithFileSystem().MakeAbsolute().ToString(SlashMode.Native)}
+            };
+        }
+        else if (programConfiguration.Sdk is LinuxClangSdk)
+        {
+            LinuxClangSdk linuxClangSdk = (LinuxClangSdk)programConfiguration.Sdk;
+
+            inputs = new[] {
+                linuxClangSdk.SysRoot,
+                linuxClangSdk.GccToolchain,
+                linuxClangSdk.ToolsPath
+            };
+
+            var compileFlags = new[] {
+                $"--sysroot={linuxClangSdk.SysRoot.ResolveWithFileSystem().MakeAbsolute().InQuotes(SlashMode.Native)}",
+                $"--gcc-toolchain={linuxClangSdk.GccToolchain.ResolveWithFileSystem().MakeAbsolute().InQuotes(SlashMode.Native)}",
+                $"-target {linuxClangSdk.TargetTriple}",
+                "-D_GLIBCXX_USE_CXX11_ABI=0"
+            };
+
+            var linkFlags = compileFlags.Concat(new[] {
+                "-fuse-ld=lld",
+                "-static-libstdc++"
+            });
+
+            commandLineArguments.Add($"CXXFLAGS='{string.Join(" ", compileFlags)}'");
+            commandLineArguments.Add($"LDFLAGS='{string.Join(" ", linkFlags)}'");
+
+            environmentVariables = new Dictionary<string, string> {
+                {"PATH", (new[] {linuxClangSdk.ToolsPath})
+                    .Concat(path)
+                    .Select(p => p.ResolveWithFileSystem().MakeAbsolute().ToString(SlashMode.Native))
+                    .SeparateWith(":")}
+            };
+        }
+        else
+        {
+            throw new Exception($"Unsupported Sdk {programConfiguration.Sdk} for {nameof(SetupLib)}");
+        }
+        
+        var buildDirectory = new NPath($"build/{targetId}_{cfg}");
+        var installDirectory = new NPath($"builds/{targetId}");
+
+        Backend.Current.AddAction(actionName: "make",
+            targetFiles: new NPath[] {},
+            inputs: inputs,
+            executableStringFor: "make",
```

Comment:
I would leave a short comment here too (maybe refer to a larger comment near the `-j` flag?)